core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -448,17 +448,18 @@ oppia.directive('conversationSkin', [
             hasInteractedAtLeastOnce = false;
             $scope.recommendedExplorationSummaries = null;
 
-            playerPositionService.init(_navigateToActiveCard);
-            oppiaPlayerService.init(function(exploration, initHtml, newParams) {
+            PlayerPositionService.init(_navigateToActiveCard);
+            ExplorationPlayerService.init(function(exploration, initHtml, 
+						   newParams) {
```

Comment:
Could you bring the whole thing from `function(exploration,....` to the next line, again no need to move it under the parenthesis, just indent it once.